app/code/Magento/InventoryApi/Test/_files/products_simple.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/** @var $product \Magento\Catalog\Model\Product */
+$product = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()
```

Comment:
currently, none of the test classes use this fixture.
As you adapted products_simple.php for your purposes
Do we really need it?